services/rapid-analysis/values-usdf-comcamsim.yaml

```diff
@@ -0,0 +1,123 @@
+namespace: rubintv-comcamsim
+image:
+  repository: lsstts/rapid-analysis
+  tag: c0035_usdf
+  pullPolicy: Always
+location: USDF
+env:
+  http_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  https_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  squid_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  no_proxy: .slac.stanford.edu
+  HTTP_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
+  HTTPS_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
+  SQUID_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
```

Comment:
I wonder why all these urls are the same. Are they changing in the future? if that's so maybe you could add a comment with a reference to a ticket which will handle that work, like a "to do".

Reply:
also, I assume it is ok that https variables don't use `https://` yet.

Reply:
Yeah it's kind of annoying to have to include all of these, but different programs look for slightly different environment variables for the proxy setting, so USDF usually sets all of them just to be sure.